walk-in-interview/src/main/webapp/new-job/script.js

```diff
@@ -0,0 +1,296 @@
+/**
+ * This file is specific to new-job.html. It renders the fields on the
```

Comment:
seems that "new-job.html" is deleted?

Reply:
I called it index.html instead, and put it in a new-job folder.